### DISCUSSION TITLE
Error in OMEdit after build

### DISCUSSION THREAD

**Discussion by ap766** [OpenModelica/OpenModelica#11700] [Q&A] [answered]
I have compiled OpenModelica from Source Code.I have followed the process for Ubuntu system.And i am running OMEdit.However I am coming accross this error.
![image](https://github.com/OpenModelica/OpenModelica/assets/79255079/4e1d9a70-e22c-4be6-830d-ce2bc990db79)

I went to the location .openmodelica/libraries and since there are so many libraries in the index.json i realised i cannot install them one by one.
![image](https://github.com/OpenModelica/OpenModelica/assets/79255079/51f6cb38-689b-4df8-b2bc-13e9a327739b)

So how do i fix the error?

**Comment by AnHeuermann** [comment, marked as answer]
Installing the libraries is a separate step and not done during compilation of OpenModelica (there is a make/cmake rule for it, though).
Use the package manager to install libraries like the Modelica standard library.
OMedit: `File`->`Manage Libraries`->`Install Library`.

**Comment by ap766** [comment]
Thanks a lot! The error is fixed.
